src/components/Checkbox/CheckboxListInput.tsx

```diff
@@ -1,15 +1,22 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 
-class CheckboxListInput extends React.Component {
+export interface CheckboxListInputProps {
+  children?: React.ReactNode;
+  className?: string;
+  onChange?: (value: string[]) => void;
+  value?: string[];
+}
+
+class CheckboxListInput extends React.Component<CheckboxListInputProps> {
   static propTypes = {
     children: PropTypes.node,
     className: PropTypes.string,
     onChange: PropTypes.func,
     value: PropTypes.array,
   };
```

Comment:
Do we still want to keep PropTypes when converting to typescript?

Reply:
Probably not worth it because it only provides console errors when running dev builds. So it's usually just noise.
I didn't spend time refactoring this file because it wasn't originally planned to be converted to TS, I was forced to due to manual .d.ts types making it into generated .d.ts files. The TS compiler doesn't know we copy .d.ts file from src to the build so it errors saying it can't link to the type.